src/lib/coda_base/user_command_payload.ml

```diff
@@ -67,8 +67,9 @@ module Common = struct
     let%map fee = Currency.Fee.gen
     and nonce = Account_nonce.gen
     and memo =
-      String.gen_with_length Memo.max_size_in_bytes Char.quickcheck_generator
-      >>| Memo.create_exn
+      String.gen_with_length Memo.max_digestible_string_length
```

Comment:
Can you quickcheck a bool and use that to determine whether to use a bytes memo or a digest memo?

Reply:
Can do.